src/_locales/en/messages.json

```diff
@@ -17,6 +17,7 @@
     "back": "Back",
     "backup": "Backup",
     "backup_persona": "Backup your persona.",
+    "backup_persona_unavailable": "Please connect persona with a social network.",
```

Comment:
```
    "camera_unavailable": "Your camera is not available.",
```

Reply:
This isn't a message when users wanna backup an unconnected persona. The camera only used when restoring persona.

Reply:
ACK